app/src/main/java/com/smashingboxes/epa_prototype_android/MainActivity.java

```diff
@@ -1,18 +1,71 @@
 package com.smashingboxes.epa_prototype_android;
 
+import android.content.Intent;
 import android.os.Bundle;
 import android.support.design.widget.FloatingActionButton;
 import android.support.design.widget.Snackbar;
 import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.Toolbar;
 import android.view.MenuItem;
 import android.view.View;
 
-import com.smashingboxes.epa_prototype_android.fitbit.FitbitLoginCache;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.smashingboxes.epa_prototype_android.fitbit.FitbitRequestManager;
+import com.smashingboxes.epa_prototype_android.fitbit.activity.Period;
+import com.smashingboxes.epa_prototype_android.fitbit.activity.TimeSeriesResourcePath;
+import com.smashingboxes.epa_prototype_android.fitbit.auth.FitbitLoginCache;
+import com.smashingboxes.epa_prototype_android.fitbit.settings.SettingsActivity;
+import com.smashingboxes.epa_prototype_android.helpers.DateHelper;
+import com.smashingboxes.epa_prototype_android.models.ActivityData;
+import com.smashingboxes.epa_prototype_android.models.FitbitProfile;
+import com.squareup.picasso.Picasso;
 
 public class MainActivity extends AppCompatActivity {
 
     private FitbitLoginCache loginCache;
+    private FitbitRequestManager requestManager;
+
+    private Picasso picasso;
```

Comment:
Not yet, but I'll remove it in a later PR if I don't end up using it to load the profile image.